### PR TITLE
add input component

### PR DESCRIPTION
**Pull request by SyedRashed007**
**_Input Component:_**

Screenshot (341)

### CODE REVIEW

**Review by MiryalaAkshitha — APPROVED** [review]

File: css/pages/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+.input-error {
+  border: 1px solid var(--red);
+}
+.msg-error {
+  color: red;
+  display: inline-block;
+  padding: 1rem;
+  font-size: 1.3rem;
+}
+.btn {
```
Comment (FarhanMobashir) [inline css/pages/Input/input.css]: Are you defining button style once again ?

Comment (SyedRashed007) [reply css/pages/Input/input.css]: It was affecting the homepage btn as well

Comment (FarhanMobashir) [reply css/pages/Input/input.css]: You should then make your button a more generic one rather defining it for every use case .

**Review by FarhanMobashir — APPROVED** [review]
Code is neat. Can be merged

**Pull request merged**